config/root/bootstrap.go

```diff
@@ -45,8 +51,28 @@ func Bootstrap(ctx context.Context, rootDiscoveryClient discovery.DiscoveryInter
 		homeWorkspaceCreatorGroupReplacement = "[]"
 	}
 
-	return confighelpers.Bootstrap(ctx, rootDiscoveryClient, rootDynamicClient, batteriesIncluded, fs, confighelpers.ReplaceOption(
+	if err := confighelpers.Bootstrap(ctx, rootDiscoveryClient, rootDynamicClient, batteriesIncluded, fs, confighelpers.ReplaceOption(
 		"HOME_CREATOR_GROUPS", homeWorkspaceCreatorGroupReplacement,
-		"HOMEPREFIX", homePrefix,
-	))
+	)); err != nil {
+		return err
+	}
+
+	// set LogicalCluster to Ready
+	return wait.PollImmediateUntilWithContext(ctx, time.Millisecond*100, func(ctx context.Context) (done bool, err error) {
+		logger := klog.FromContext(ctx).WithValues("bootstrapping", "root-phase0")
+		this, err := kcpClient.CoreV1alpha1().LogicalClusters().Get(ctx, corev1alpha1.LogicalClusterName, metav1.GetOptions{})
+		if err != nil {
+			logger.Error(err, "failed to get this workspace in root")
+			return false, nil
+		}
+		if this.Status.Phase == corev1alpha1.LogicalClusterPhaseInitializing {
+			this.Status.Phase = corev1alpha1.LogicalClusterPhaseReady
+			_, err = kcpClient.CoreV1alpha1().LogicalClusters().UpdateStatus(ctx, this, metav1.UpdateOptions{})
+			if err != nil {
+				logger.Error(err, "failed to update this workspace status in root")
```

Comment:
`failed to update status for logicalcluster root|cluster`?

Reply:
removed this one. phase-0 does this already.